sum.py

```diff
@@ -0,0 +1,7 @@
+def sum(args):
+    sum_of_args = 0
+    for i in args:
+        sum_of_args += i
+    return sum_of_args
+
+print(sum([2, 3]))
```

Comment:
- Use intermediate variables, plz see my fixes for js example and refactor py example